scripts/apiStage.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+#
+#
+# Quick check to see if a container is running
+#
+checkContainerRunning()
+{
+  local CHECKONE=$1
+  local COUNT=1
+  local DURATION=10
+  local DELAY=0.1
+
+  printf "Check container running..."
+
+  while [ $COUNT -le $DURATION ]; do
+    sleep $DELAY
+    (( COUNT++ ))
+    if [ -z "$($PODMAN ps | grep $CHECKONE)" ]; then
+      break
+    fi
+  done
+
+  if [ ! -z "$($PODMAN ps | grep $CHECKONE)" ]; then
+    printf "${GREEN}Container SUCCESS"
+    printf "\n\n${NOCOLOR}"
+  else
+    printf "${RED}Container ERROR"
+    printf "\n\n  Error: ${RED}Check container \"${CHECKONE}\""
+    printf "${NOCOLOR}\n"
+  fi
+}
+#
+#
+# Clean podman/docker
+#
+cleanApi()
+{
+  echo "Cleaning everything, \$PODMAN and data..."
+  printf "${RED}\n"
+  $PODMAN system prune -f
+  printf "${GREEN}$PODMAN cleaning success.${NOCOLOR}\n"
+}
+#
+#
+# Stop all QPC containers
+#
+stopApi()
+{
+  $PODMAN stop -t 0 $($PODMAN ps --filter name="qpc*")
```

Comment:
The command below treats the entire output as one block, which helps keep the grouping of container names or IDs intact, especially important if they include spaces.

`$PODMAN stop -t 0 "$($PODMAN ps --filter name='qpc*')"`